simple-semigroupoids/README.md

```diff
@@ -0,0 +1 @@
+# simple-semigroupoids
```

Comment:
Perhaps worth adding some information here?

Reply:
I plan on doing that at a later stage, but definitely before releasing the package on CHaP!